public/index.html

```diff
@@ -570,146 +574,156 @@
+        </div>
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-4 ppe-example">
+              <img style="height:140px"src="images/goggles.png" alt="example of personal protective equipment: goggles"/>
+              <div class="ppe-example-caption i18n" data-i18n="ftm-item-goggles">goggles</div>
+            </div>
+            <div class="col-lg-4 ppe-example">
+              <img style="height:210px" src="images/masks.png" alt="example of personal protective equipment: surgical mask, N95"/>
+              <div class="ppe-example-caption i18n" data-i18n="ftm-item-masks">Surgical mask & N95 Masks</div>
+            </div>
+            <div class="col-lg-4 ppe-example">
+              <img style="height:185px" src="images/faceshield.png" alt="example of personal protective equipment: face shield "/>
+              <div class="ppe-example-caption i18n" data-i18n="ftm-item-face-shields">Face Shield</div>
+            </div>
           </div>
         </div>
-      </div>
 
-      <div class="content-container" style="margin-bottom: 40px;">
-        <h2>Protecting healthcare workers protects everybody</h2>
-
-        <p class="i18n" data-i18n="ftm-index-where-to-find-ppe">
-          Maybe you bought N95 masks a few weeks ago or already have some in your house. Right now healthcare workers are running dangerously low on PPE like masks, hospital gowns, booties, and face shields to protect them while they risk their lives to save others.
-        <p>
-        </p >
-          By donating, you protect <em>everybody</em>: doctors, nurses, hospital staff, patients, and the general public.
-        <p>
-        </p class="i18n" data-i18n="[html]ftm-index-types-of-ppe">
-          If you know construction workers, vets, scientists, or dentists, please share this link with them.
-        </p>
-
-        <hr/>
-
-        <h2 class="i18n" data-i18n="ftm-add-donation-site-header">Do you need PPE?</h2>
-        <div class="need-ppe">
-          <div class="i18n" data-i18n="ftm-add-donation-site-question"> Want to add a donation site? update or remove information?</div>
-          <div class="i18n" data-i18n="[html]ftm-add-donation-site-answer">
-            Please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">
-            fill out this form</a>. We will review it as soon as possible.
+        <div class="content-container" style="margin-bottom: 40px;">
+          <h2 class="i18n" data-i18n="ftm-protecting-healthcare-workers">Protecting healthcare workers protects everybody</h2>
+
+          <p class="i18n" data-i18n="ftm-index-where-to-find-ppe">
+            Maybe you bought N95 masks a few weeks ago or already have some in your house. Right now healthcare workers are running dangerously low on PPE like masks, hospital gowns, booties, and face shields to protect them while they risk their lives to save others.
+          <p>
```

Comment:
I think this was already there but open/close tags look a little off

Reply:
wow wtf :)